src/util-privs.c

```diff
@@ -136,6 +149,84 @@ void SCDropCaps(ThreadVars *tv)
 #endif
 }
 
+#define SCPrivsRaise(...)
+#define SCPrivsDrop(...)
+#define _SCPrivsDrop(...)
+
+#else /* HAVE_LIBCAP_NG */
+
+static void SCSetGroupID(const uint32_t gid)
+{
+    int ret = setegid(gid);
+
+    if (ret != 0) {
+        SCLogError(SC_ERR_GID_FAILED, "unable to set the group ID,"
+                " check permissions!! gid=%u ret=%i errno=%i", gid, ret, errno);
```

Comment:
We can use ```FatalError``` instead of SCLogError+exit.

Also I think its nicer to print the error string ```strerror(errno)``` than the numeric code.